### DISCUSSION TITLE
Kafka topic retention.bytes issue

### DISCUSSION THREAD

**Discussion by Zaven96** [strimzi/strimzi-kafka-operator#6209] [Q&A]
Kafka topic config is set retention.bytes: 75161927680 but I detected in my monitoring that the size of the partitions are not limited to what size I set.
<img width="812" alt="Screen Shot 2022-01-17 at 16 59 57" src="https://user-images.githubusercontent.com/46746797/149773616-4e71ea25-668b-4586-824a-0d963cff308b.png">
As you can see on the screen the partition size is 179Gb but my config set 75Gb.
what can I do?

**Comment by ppatierno** [comment]
What's the metric you are using to draw this Grafana graph? It's just a single partition or all partitions making your topic?

**Reply by Zaven96** [reply]
strimzi kafka prometheus metrics, all partitions.
interesting thing retention.ms working.
